mvnw

```diff
@@ -200,6 +200,42 @@ if [ -z "$BASE_DIR" ]; then
   exit 1;
 fi
 
+##########################################################################################
+# Extension to allow automatically downloading the maven-wrapper.jar from Maven-central
+# This allows using the maven wrapper in projects that prohibit checking in binary data.
+##########################################################################################
+JAR_URL="http://central.maven.org/maven2/io/takari/maven-wrapper/0.2.1/maven-wrapper-0.2.1.jar"
+if [ -r "$BASE_DIR/.mvn/wrapper/maven-wrapper.jar" ]; then
+    echo "Found .mvn/wrapper/maven-wrapper.jar"
+else
+    echo "Couldn't find .mvn/wrapper/maven-wrapper.jar, downloading it ..."
+    if command -v wget > /dev/null; then
+        echo "Found wget ... using wget"
+        wget ‐‐output-document=.mvn/wrapper/maven-wrapper.jar "$JAR_URL"
```

Comment:
in some environment, such as in alpine, wget has no ‐‐output-document option, only '-O'  works.